dom/nodes/insertion-effects/Node-appendChild-script-and-custom-from-fragment.tentative.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Node.appendChild: inserting script and custom element from a DocumentFragment</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id="log"></div>
+<script>
+let customConstructed = false;
+let isCustomConstructed = false;
```

Comment:
nit: Maybe make this a more descriptive variable name like `customConstructedDuringEarlierScript` or something, just to indicate that it's a snapshot of the above variable during an earlier script's execution.